addons/godot-xr-tools/objects/grab_points/grab.gd

```diff
@@ -229,7 +226,6 @@ static func _remove_collision_exceptions( \
 
 	# For now we'll remove all.
 
-	for body : PhysicsBody3D in exceptions:
-		if is_instance_valid(body):
-			on_collision_hand.remove_collision_exception_with(body)
-			body.remove_collision_exception_with(on_collision_hand)
+	for body : RID in exceptions:
```

Comment:
If we are going to remove the collisions directly on the physics server, we should also add them this way.